### PR TITLE
Updated github-git-cheat-sheet.md

### LINKED ISSUE #759: Is there something wrong with description of "git  log --follow [file]"?
## Overview
There is something wrong with description of "git  log --follow [file]".

#### Reasoning
the `--follow` option in [documentation of Git](https://git-scm.com/docs/git-log) :
image

But the meaning of the `--follow` in this repository is inverse:
image

### PR DESCRIPTION
**Pull request by mainak-debnath**
Updated definition of git log --follow

## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>
The definition of `git log --follow` in the github git-cheat-sheet.md was different than the actual meaning. This pull request updates that definition.
<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

closes #759

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

### CODE REVIEW

**Comment by brianamarie** [comment]
Thank you for this clarification, @mainak-debnath!
